uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -599,6 +639,7 @@ define([
       this._init = function () {
         var self = this;
 
+        this.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes).join(', ');
```

Comment:
This should be an array, in my opinion. The joining via `,` is a presentational thing, hence it should be done in the view